src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -4391,6 +4468,29 @@ int wm_vuldet_oval_xml_parser(OS_XML *xml, XML_NODE node, wm_vuldet_db *parsed_o
                         not_vulnerable = "while related to the CVE in some way, a decision has been made to ignore this issue";
                     }
 
+                    // In ALMA 8 some architectures are in the criterion tag
+                    if (dist_ref == FEED_ALMA8 && strstr(node[i]->values[j], " arch is ")) {                        
```

Comment:
Can it be the case where these architectures are in the `criterion` for other _AlmaLinux_ feeds?
If so, the condition would have to be changed for all _AlmaLinux_ feeds.
If this is not the case, then this would be fine.